server/odc-service/src/main/java/com/oceanbase/odc/service/iam/ResourceRoleService.java

```diff
@@ -125,8 +126,10 @@ public Map<Long, Set<ResourceRoleName>> getProjectId2ResourceRoleNames() {
 
     @SkipAuthorize
     public Map<Long, Set<ResourceRoleName>> getProjectId2ResourceRoleNames(Long userId) {
-        Map<Long, ResourceRole> id2ResourceRoles = listResourceRoles().stream().collect(Collectors
-                .toMap(ResourceRole::getId, resourceRole -> resourceRole, (existingValue, newValue) -> newValue));
+        Map<Long, ResourceRole> id2ResourceRoles = resourceRoleRepository.findAll().stream()
```

Comment:
You may need filter database resource role.

Reply:
I don't quite understand your intention. In this part, the role to be queried is controlled by the parameters passed in.

Reply:
I mean `resourceRoleRepository.findAll()` will query the database resource role, but this method `getProjectId2ResourceRoleNames` only deal with project resource role.